server/client.go

```diff
@@ -3192,10 +3225,40 @@ func (c *client) Account() *Account {
 // prunePerAccountCache will prune off a random number of cache entries.
 func (c *client) prunePerAccountCache() {
 	n := 0
-	for cacheKey := range c.in.pacache {
-		delete(c.in.pacache, cacheKey)
-		if n++; n > prunePerAccountCacheSize {
-			break
+	if time.Now().UnixNano()-c.in.losc >= int64(orphanSubsCheckInterval) {
```

Comment:
Make simple time.Time?

Reply:
24 bytes... I thought we were trying to put things on a diet ;-)

Reply:
hmm, good point.

Reply:
So keeping the way it is then?

Reply:
I think just Unix() no? Why Nano?

Reply:
so that we can use less than 1sec.. Remember the jwt expiration tests flapping tests? I can do it, but then understand that test around this will have to be slower.

Reply:
I understand, but I don't think setting it sub-second make sense which this suggest we can do. But we only do it for tests.